libs/utils/analysis/latency_analysis.py

```diff
@@ -49,6 +51,26 @@ def __init__(self, trace):
 
     @memoized
     def _dfg_latency_df(self, task):
+        """
+        DataFrame of task's wakeup/suspend events
+
+        The returned DataFrame has these columns
+        - Time: the time an event related to this task happened
```

Comment:
Well... because you know Pandas internals... but to the "average" user, that's nothing else that another column. I just want to describe what the timstamp represents, in this case it's "just" the task events... but in other cases, like the ones below, it's a "generated time" which corresponds explicitly the wakup or blocking time.

Perhaps I can specify that this is also the DF's index.